api/apps/articles/model.coffee

```diff
@@ -206,7 +211,10 @@ sortParamToQuery = (input) ->
           featured_artist_ids: article.featured_artist_ids.map(ObjectId) if article.featured_artist_ids
           featured_artwork_ids: article.featured_artwork_ids.map(ObjectId) if article.featured_artwork_ids
           biography_for_artist_id: ObjectId(article.biography_for_artist_id) if article.biography_for_artist_id
-          contributing_authors: article.contributing_authors.map(ObjectId) if article.contributing_authors
+          contributing_authors: article.contributing_authors.map( (id)->
```

Comment:
I think it would be clearer to name this `(author) ->` and `author.id = ObjectId(author.id)`